src/controller/balance-controller.ts

```diff
@@ -95,6 +95,10 @@ export default class BalanceController extends BaseController {
    * @param {string} date.query - Timestamp to get balances for
    * @param {integer} minBalance.query - Minimum balance
    * @param {integer} maxBalance.query - Maximum balance
+   * @param {boolean} hasFine.query - Only users with(out) fines
+   * @param {integer} minFine.query - Minimum fine
+   * @param {integer} maxFine.query - Maximum fine
+   * @param {Array<string>} userTypes.query - User types (strings, not numbers)
```

Comment:
Mis handig om bij te zetten wat de opties zijn? dan zie je dat ook terug in de docs + frontend client.

Reply:
Resolved in cfcbd191b85b3707a4b3e2321c59065dc4a95ea6